src/main/java/com/cloudbees/dockerpublish/DockerBuilder.java

```diff
@@ -65,14 +65,16 @@
     private boolean skipDecorate;
     @CheckForNull
     private String repoTag;
+    private String retryBuild;
     private boolean skipPush = true;
     private boolean createFingerprint = true;
     private boolean skipTagLatest;
 
     @Deprecated
-    public DockerBuilder(String repoName, String repoTag, boolean skipPush, boolean noCache, boolean forcePull, boolean skipBuild, boolean skipDecorate, boolean skipTagLatest, String dockerfilePath) {
+    public DockerBuilder(String repoName, String repoTag, String retryBuild, boolean skipPush, boolean noCache, boolean forcePull, boolean skipBuild, boolean skipDecorate, boolean skipTagLatest, String dockerfilePath) {
```

Comment:
don't change the signature of the constructor, not needed with DataBoundSetter below